gp-nested-forms/gpnf-triggered-population.php

```diff
@@ -150,6 +165,7 @@ public function is_applicable_form( $form ) {
 		return empty( $this->_args['form_id'] ) || $form_id == $this->_args['form_id'];
 	}
 
+
```

Comment:
Ideally, whitespace changes (additions/removals) aren't snuck in unless it's fixing a code style. I won't hold up the PR over this! Just something to be aware of.

Reply:
Got it! Will remove :+1: